### ISSUE TITLE
[$250] Add back the parallax animation in the new arch

### ISSUE THREAD

**Issue by roryabraham** [Expensify/App#40309]
### Problem

The gyroscope parallax animation was causing serious performance issues on the new arch, so we reverted it in https://github.com/Expensify/App/pull/40302

### Solution

Address the root cause of the problem and add the animation back.
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0103e0b53e212479af</li>
        <li>Upwork Job ID: 1795458292540534784</li>
        <li>Last Price Increase: 2024-05-28</li>
<li>Automatic offers: </li>
<ul>
<li>hungvu193 | Contributor | 102495616</li>
</ul></li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @hoangzinh</details>

**Comment by roryabraham** [comment]
cc @WoLewicki

**Comment by WoLewicki** [comment]
Commenting

**Comment by roryabraham** [comment]
@WoLewicki any update here? I wonder if that the performance problem with the parallax animation and live markdown might actually be indicative of a [broader performance regression with the new arch](https://expensify.slack.com/archives/C05LX9D6E07/p1714768240955059)

**Comment by WoLewicki** [comment]
Yeah, the PR is there: https://github.com/Expensify/App/pull/41103. Sorry, I forgot to link it here. As for the regression, there still might be points where there are regressions in performance connected to the new arch and we are trying to find them and resolve each time we see any. Also, for some reason I cannot open the link properly though 😕

**Comment by roryabraham** [comment]
This was done in two places:

- https://github.com/Expensify/App/pull/41103
- https://github.com/Expensify/App/pull/41695

**Comment by hungvu193** [comment]
Hey @roryabraham , since I reviewed https://github.com/Expensify/App/pull/41103, can you reopen it to handle the payment?

**Comment by lschurr** [comment]
Great, all set! Payment sent to @hungvu193 in Upwork.
